File: CS/MapControl_BingMapsDataProvider/Form1.cs
using System.Windows.Forms;
using DevExpress.XtraMap;

namespace MapControl_BingMapsDataProvider {
    public partial class Form1 : Form {
        public Form1() {
            InitializeComponent();
            #region #BingMapsDataProvider
            mapControl1.Layers.Add(new ImageLayer() {
                DataProvider = new BingMapDataProvider() {
                    BingKey = "YOUR BING KEY",
                    Kind = BingMapKind.Road
                }
            });
            #endregion #BingMapsDataProvider
        }
    }
}

File: VB/MapControl_BingMapsDataProvider/Form1.vb
Imports System.Windows.Forms
Imports DevExpress.XtraMap

Namespace MapControl_BingMapsDataProvider
	Partial Public Class Form1
		Inherits Form

		Public Sub New()
			InitializeComponent()
'			#Region "#BingMapsDataProvider"
			mapControl1.Layers.Add(New ImageLayer() With {
				.DataProvider = New BingMapDataProvider() With {
					.BingKey = "YOUR BING KEY",
					.Kind = BingMapKind.Road
				}
			})
'			#End Region ' #BingMapsDataProvider
		End Sub
	End Class
End Namespace
